modules/notification/webhook/webhook.go

```diff
@@ -766,12 +766,12 @@ func sendReleaseHook(doer *models.User, rel *models.Release, action api.HookRele
 		return
 	}
 
-	mode, _ := models.AccessLevel(rel.Publisher, rel.Repo)
+	mode, _ := models.AccessLevel(doer, rel.Repo)
 	if err := webhook_services.PrepareWebhooks(rel.Repo, models.HookEventRelease, &api.ReleasePayload{
 		Action:     action,
 		Release:    convert.ToRelease(rel),
 		Repository: convert.ToRepo(rel.Repo, mode),
-		Sender:     convert.ToUser(rel.Publisher, nil),
+		Sender:     convert.ToUser(doer, nil),
```

Comment:
```suggestion
		Sender:     convert.ToUser(doer, doer),
```

Reply:
that should also be fixed on main ...

Reply:
#17820

Reply:
Wait actually... I had a quick look at the other usages, they all use `nil` and not `doer`.

Reply:
well could call it a bug - @Gusted nil should only be used if doer is not known